### ISSUE TITLE
Adding collections directly

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/pelican-frontend#44]
I would like to be able to add collections to Pelican directly without needing to send an email, so that we can turn around data reviews more quickly.

**Comment by jpmckinney** [comment]
Previously, a message had to be sent to RabbitMQ (typially via its admin interface, as otherwise setting up a RabbitMQ channel requires Python code). Now, an API endpoint was added for integration with the data registry.

I think we can now do something like:

```
curl -d '{"name":"spider_data_version","collection_id":123}' -H 'Content-Type: application/json' http://pelican.api/api/dataset_start
```

This step would then be similar to how we schedule spiders. How's that?

**Comment by duncandewhurst** [comment]
Sounds good to me. We can document it in the deploy repo and the process note on preparing data for analysis.

Can we also prepare some brief documentation on what success looks like (collection appears in the list on https://dqt.datlab.eu/?) and any error messages that might be encountered?

**Comment by jpmckinney** [comment]
Yes, success looks like it appearing in the list, with the status column otherwise indicating the progress.

We will have to add the error messages over time, as I'm not sure which currently occur (but feel free to share some descriptions in this issue, as I'll move this to the deploy repo for documentation).

I'll be deploying Pelican on the Helpdesk server, probably pelican.kingfisher.open-contracting.org (in case we enable the pelican frontend on the registry, which would be pelican.data.open-contracting.org).
